README.md

```diff
@@ -18,7 +18,7 @@ All docs and design specs for our client-server based chat application to rule t
 * GetUsers
 * GetChannel
 * Auth
-* GetUserDetails
+* [GetUserDetails](https://github.com/TI114/chatserver_docs/blob/getUserDetails/getuserdetails.md)
```

Comment:
please use a relative link to the file:

`[getUserDetails](getuserdetails.md)`

This will link to a file called getuserdetails.md in the same directory as this file (README.md)